pages/models.py

```diff
@@ -2452,13 +2451,10 @@ class TutorMarketing(Page):
     cost_institution_message = models.CharField(max_length=255)
 
     #feedback
-    feedback_image = models.ForeignKey(
-        'wagtailimages.Image',
-        on_delete=models.SET_NULL,
-        null=True,
-        blank=True,
-        related_name='+',
-    )
+    feedback_video = StreamField(
```

Comment:
According to the card acceptance criteria, a video or an image can be used on this page.

You can just add an `ImageBlock` to this stream field - with the `max_num=1`, it will take either a video or image. You might consider renaming the field though.

Reply:
@mwharrison ImageBlock has been added